autopts/wid/bass.py

```diff
@@ -214,6 +215,34 @@ def hdl_wid_116(_: WIDParams):
     Please expose a Broadcast Receive State with BIG_Encryption
     field indicating Broadcast_Code required.
     """
+
+    stack = get_stack()
+
+    # Add a Broadcast Receive State with BIG_Encryption field set to indicate
+    # Broadcast_Code required (as per test requirement for BASS/SR/CP/BV-21-C)
```

Comment:
```suggestion
    # Broadcast_Code required
```

The WID handler perhaps shouldn't be considered unique for this test